src/SFA.DAS.EmployerAccounts.Web/Views/Home/Index.cshtml

```diff
@@ -8,72 +8,76 @@
 @{ ViewBag.HideNav = true; }
 @{ ViewBag.HideAccounts = "true"; }
 @{ ViewBag.GaData.Vpv = "/page-auth-homepage"; }
+@{Layout = "~/Views/Shared/_Layout_CDN.cshtml"; }
 
 @if (!string.IsNullOrWhiteSpace(Model.Data.ErrorMessage))
 {
-    <div class="grid-row">
-        <div class="column-full">
-            <div class="panel panel-error">
-                @Model.Data.ErrorMessage
-            </div>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <div class="panel panel-error">

```

Comment:
.panel and .panel-error don't exist in the CDN, should these be updated?

Reply:
My excuse for this one is that no errors are fired on this page (and it appears - as confirmed by Chris - that this is/was rouge code in the file). This redundant "error message HTML" has now been removed completely.